tools/NuGet/template-nuget/DynamoVisualProgramming.Core.nuspec

```diff
@@ -27,6 +27,7 @@
                 <dependency id="NUnit" version="2.6.3"/>
                 <dependency id="DynamoVisualProgramming.ZeroTouchLibrary" version="$Version$"/>
                 <dependency id="Newtonsoft.Json" version="8.0.3"/>
+				<dependency id="System.Collections.Immutable" version="1.7.1"/>
```

Comment:
What's with the extra whitespace?

Reply:
Hmmm...it's strange because I still don't see the extra spacing in my IDE. I just used the XML formatter in VS Code, then it just updated somehow.